File: wrappers/dotnet/indy-sdk-dotnet/WalletApi/WalletEncryptionException.cs
namespace Hyperledger.Indy.WalletApi
{
    /// <summary>
    /// Exception thrown when an error occurred during encryption-related operations.
    /// </summary>
    public class WalletEncryptionException : IndyException
    {
        const string message = "An error occurred during encryption-related operations.";

        /// <summary>
        /// Initializes a new WalletEncryptionException.
        /// </summary>
        internal WalletEncryptionException() : base(message, (int)ErrorCode.WalletEncryptionError)
        {

        }
    }

}

File: wrappers/java/src/main/java/org/hyperledger/indy/sdk/wallet/WalletEncryptionException.java
package org.hyperledger.indy.sdk.wallet;

import org.hyperledger.indy.sdk.ErrorCode;
import org.hyperledger.indy.sdk.IndyException;

/**
 * Exception thrown occurred during encryption-related operations.
 */
public class WalletEncryptionException extends IndyException
{
	private static final long serialVersionUID = 1829076830401150667L;
	private final static String message = "Error during encryption-related operations.";

	/**
	 * Initializes a new WalletEncryptionException.
	 */
	public WalletEncryptionException()
	{
		super(message, ErrorCode.WalletEncryptionError.value());
	}
}